packages/financial-templates-lib/src/price-feed/UniswapPriceFeed.js

```diff
@@ -76,6 +76,15 @@ class UniswapPriceFeed extends PriceFeedInterface {
     }
   }
 
+  getHistoricalPricePeriods() {
```

Comment:
this looks good to me, but may need to worry about how this works with the medianizer